docs/guides/create-webshop.md

```diff
@@ -0,0 +1,208 @@
+Having Products and being able to browse them is great and all, but customers will also need a way to checkout and pay for their cart's content. This is where payment providers come into the picture. For simplicity's sake, we'll use Stripe as the payment provider here. Ensure the **Orchard Core Commerce - Payment - Stripe** feature is enabled, then go to _Configuration > Commerce > Stripe API_. The Publishable Key and Secret Key fields need to be filled in for the Stripe Payment form to work on the checkout page, see the links on the settings page for more.
+
+![Stripe settings.](../assets/images/create-webshop/step-6/stripe-settings.png)
+
+_If you don't already have the necessary API keys, follow the links._
+
+With that done, the Stripe Payment form now appears on the checkout page, making checking out possible.
+
+![Stripe Payment.](../assets/images/create-webshop/step-6/stripe-form.png)
+
+_This will allow customers to pay to their card's content._
+
+## Other useful Commerce features
+
+### Order management
+Orders contain all data related to the transaction that triggered their creation. Just like most things in Orchard, Orders are content items, and they are automatically created when a user checks out. They can also be created manually on the dashboard.
+
+![Orders are created on checkout.](../assets/images/create-webshop/order-management/checkout-order-creation.png)
+
+_Orders include all sorts of information about the transaction, including charges._
+
+To create Orders manually, head to the usual _Content > Content Items_ page and select Order from the New button's dropdown menu.
+
+![Orders can be created manually.](../assets/images/create-webshop/order-management/create-order-manually.png)
+
+_Products can be added to and removed from any Order manually._
+
+### Price Variants Products
+Products that have `PriceVariantsPart` attached to them can have multiple different prices associated with them based on predefined attributes. This is handy when there are variants of the given product and you want to set different prices for each version. These prices are generated based on those Text Product Attribute Fields of the content type that are restricted to predefined values. For a more technical overview, see the [part's documentation](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/docs/features/price-variants-part.md).
+
+Create a new product type, or edit an existing one, and attach `PriceVariantsPart` to it. Then add a new Text Product Attribute Field to it.
+
+![Text attribute addition.](../assets/images/create-webshop/price-variants-products/add-text-attribute-field.png)
+
+_Name the field at will, but make sure to choose the right field type._
+
+Next, edit the newly added field's settings, fill out the Predefined values box and tick the Restrict to predefined values checkbox.
+
+![Text attribute editing.](../assets/images/create-webshop/price-variants-products/edit-text-attribute-field.png)
```

Comment:
Out of curiosity, is there a significance or just a random pick of 3 colors?

Reply:
It's the color scheme of the company vehicle I use for daily rapid commute:
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/51411356/1a18db31-d05f-4eed-9ce6-c0f877eead6a)